docs/exposition.rst

```diff
@@ -161,6 +161,18 @@ string::
 
 :ref:`Read more <set-fields>`
 
+Enum Fields
+-----------
+
+You can take advantage of MySQL's ``ENUM`` type through the ``EnumField``
+provided by django-mysql. Using ``ENUM`` provides the benefit of storing
```

Comment:
```
Using a ``CharField`` with a limited set of strings leads to inefficient data storage since the string value must be stored over and over. MySQL's ``ENUM`` type allows a more compact representation of such columns by storing the list of strings just once and using integers in the data rows, which ``EnumField`` allows you to use conveniently::
```